rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -350,6 +350,10 @@ then
 %endif
   CFRUDDER_FIRST_INSTALL=1
 fi
+# mandatory with systemd wrapper for old init
+%if 0%{?suse_version} && 0%{?suse_version} >= 1315
```

Comment:
This is sles12 and later, @amousset dou you think the condition should be larger ?

Reply:
isn't it something we should do on other distro as well (debian9, rhel 7) ?

Reply:
It it tied to how chkconfig --add and update-rc.d is done by the distro (this command should reload by itself). So this is distro specific and it has not been reported on other distro yet.
That's why i'm asking for comment from @amousset